ParisBNF/abb/BNFabb106.xml

```diff
@@ -0,0 +1,226 @@
+                                            <num value="3">፫</num>
+                                            <dim unit="leaf">10</dim>
+                                            <locus from="21r" to="30v"/>
+                                        </item>
+                                        <item xml:id="q5" n="4">
+                                            <num value="4">፬</num>
+                                            <dim unit="leaf">10</dim>
+                                            <locus from="31r" to="40v"/>
+                                        </item>
+                                        <item xml:id="q6" n="5">
+                                            <num value="5">፭</num>
+                                            <dim unit="leaf">10</dim>
+                                            <locus from="41r" to="50v"/>                                    
+                                        </item>
+                                        <item xml:id="q7" n="6">                                            
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="51r" to="58v"/> 
+                                        </item>  
+                                    </list>
+                                </collation>
+                                <condition key="good"/>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="3" writtenLines="24">   
+                                    <ab type="ruling" subtype="pattern" cert="medium">1A-1A-1A1A-1A1A/0-0/0-0/C</ab>
+                                    <ab type="ruling">Ruling is partly visible. </ab>
+                                    <ab type="pricking"> Pricking is partly visible.</ab>
+                                    <ab type="ruling" cert="unknown"> The upper line is written above the ruling. </ab>
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <seg type="script">19th-century script</seg> 
+                                <desc> <persName
+                                    ref="PRS1071dAbbadi">A. d'Abbadie</persName> describes the handwriting as ʻneatʼ<foreign xml:lang="fr">(ʻfineʼ)</foreign>.    
+                                </desc>
+                            </handNote>
+                        </handDesc>
+                            <additions>
+                            <list>
+                                <item xml:id="e1">
+                                    <desc type="StampExlibris">
+                                        Shelfmark of the <placeName ref="INS0303BNF">Library</placeName> glued onto the front board.
```

Comment:
Either write out which library or omit the word Library all together, then the title will be taken from INS0303BNF

the way you have encoded the reader will not see which Library you are talking about, and since some manuscripts may have been stamped by different libraries this is confusing

So it should be 
either
```
<desc type="StampExlibris">
                                        Shelfmark of the <placeName ref="INS0303BNF">French National Library</placeName>
```

or 
```
<desc type="StampExlibris">
                                        Shelfmark of the <placeName ref="INS0303BNF"/>
```

I personally would prefer either an empty tag OR the exact wording of the stamp

Reply:
(here and everywhere in similar cases)